src/client/components/pages/parts/collections-table.js

```diff
@@ -91,13 +91,32 @@ class CollectionsTable extends React.Component {
 		// 2.Check if user is logged In if so checks the page is central public collections page or not
 		const showNewCollectionButton = (user && user.id === ownerId) ||
 		(user && !ownerId);
+
+		const myCollectionButton = (
+			<Button
+				bsStyle="success"
+				className="margin-bottom-d5"
+				href={`/editor/${user.id}/collections`}
+				type="button"
+			>
+				&nbsp;My Collections
```

Comment:
```suggestion
				My Collections
```

Since there's no icon before the text we don't need that space

Reply:
I have made the suggested changes.